### ISSUE TITLE
Evaluation on nuscenes test set

### ISSUE THREAD

**Issue by evgeniishch** [mit-han-lab/bevfusion#317]
Could you please provide a script or information on how should one use tools/test.py to evaluate a pretrained camera+lidar detection model on NuScenes test set? 
Best wishes!

**Comment by kentang-mit** [comment]
This would require some modification to the current codebase. Basically you need to make sure that the pkl info file for the test set is loaded.  The command to create a test submission will look like the following one:

```bash
torchpack dist-run -np 8 python tools/test.py [path to config file] [checkpoint file] --format-only --eval-options 'jsonfile_prefix=[prediction directory]'
```

**Comment by youngfly** [comment]
Dear author, we are curious about "This would require some modification to the current codebase"?what should we do if we want to test on the testing set?
Best wishes?

**Comment by kentang-mit** [comment]
To name a few, you may need to combine the info files for training and validation dataset to a single one, and avoid loading annotations on the test set (because there is no annotation for v1.0-test).

**Comment by kentang-mit** [comment]
Closed due to inactivity. Please feel free to reopen if you feel it necessary.
